src/components/header/desktop-header-content.tsx

```diff
@@ -0,0 +1,315 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Link } from "react-router-dom";
+import styled, { css } from "styled-components";
+
+import { Theme } from "../../utils/enums";
+
+import DarkModeLightIcon from "../../../public/icons/dark-mode-light.svg";
+import DarkModeDarkIcon from "../../../public/icons/dark-mode-dark.svg";
+
+import LightModeLightIcon from "../../../public/icons/light-mode-light.svg";
+import LightModeDarkIcon from "../../../public/icons/light-mode-dark.svg";
+
+import { useClickOutside } from "../../utils/hooks/use-click-outside-hook";
+import {
+  ActiveProps,
+  CompareButtonProps,
+  GithubImage,
+  MenuProps,
+} from "./common";
+
+const GITHUB_LINK = "https://github.com/visgl/loaders.gl-showcases";
```

Comment:
Move this link to separate file. It is duplicated in non-desktop-header file.

Reply:
Done